.github/workflows/release.yml

```diff
@@ -103,6 +103,8 @@ jobs:
           path: src/github.com/Microsoft/hcsshim
 
       - name: Make
+        env:
```

Comment:
Not sure if more changes are needed in the `release.yml` though (wasn't sure how to test/verify this)

Reply:
I usually manually test `release.yml` changes by pushing a tag to my fork, I can test this one